tests/test_voc_format.py

```diff
@@ -729,6 +729,27 @@ def test_can_report_missing_field_in_classification(self):
             with self.assertRaisesRegex(InvalidAnnotationError, "invalid number of fields"):
                 Dataset.import_from(test_dir, format="voc_classification").init_cache()
 
+    @mark_requirement(Requirements.DATUM_ERROR_REPORTING)
+    def test_can_parse_classification_without_errors(self):
+        with TestDir() as test_dir:
+            subset_file = osp.join(test_dir, "ImageSets", "Main", "test.txt")
+            os.makedirs(osp.dirname(subset_file))
+            with open(subset_file, "w") as f:
+                f.write("a\n")
+                f.write("b\n")
+                f.write("c\n")
+
+            ann_file = osp.join(test_dir, "ImageSets", "Main", "cat_test.txt")
+            with open(ann_file, "w") as f:
+                f.write("a -1\n")
+                f.write("b 0\n")
+                f.write("c 1\n")
+
+            parsed = Dataset.import_from(test_dir, format="voc_classification")
+            parsed.init_cache()
+
+            self.assertEqual(len(parsed), 3)
```

Comment:
Perhaps you should check that the annotations are loaded correctly? That is, that 0 is indeed interpreted as the class not being present.

Reply:
Added